phpstan.json

```diff
@@ -0,0 +1,3 @@
+{
+  "defaultLevel": 6
+}
```

Comment:
newline is missing here

Reply:
if it's missing, there gonna be a special sign in red.